src/sentry/lang/java/plugin.py

```diff
@@ -169,6 +170,56 @@ def close(self):
         for archive in self._archives:
             archive.close()
 
+        # Symbolicator/Python A/B testing
+        if symbolicator_stacktraces := self.data.pop("symbolicator_stacktraces", None):
```

Comment:
please move all this into a separate function, and try/except it

Reply:
Not sure what I could catch here though; I don't know how the A/B test could go wrong and I'm not allowed to put a blanket `except`.